package.json

```diff
@@ -5,6 +5,9 @@
     "url": "https://github.com/cardstack/boxel-motion"
   },
   "scripts": {
+    "deploy:boxel-host": "cd packages/host && NODE_OPTIONS='--max_old_space_size=4096' pnpm exec ember deploy",
```

Comment:
I think (?) the max_old_space_size is only necessary when building for a fastboot server too. we can use `FASTBOOT_DISABLED=true` env var to not do the fastboot build, which i think means you wont have to increase the heap size. not doing the fastboot build should still be ok for the card prerendering needs of the realm server. it just needs the conventional build's dist artifacts to work.

Reply:
I’ll try that, thanks

Reply:
argh, i guess the heap error is not tied to the fastboot build, but rather its tied to performing a production build. sorry about that. i guess we need to keep the max_old_space_size in there. You can still leave in `FASTBOOT_DISABLED=true`, though, as that will result in a quicker build.

Reply:
or wait, i'm sorry, i think it will actually do nothing since we are doing a build only and not serving. (i'm getting building conflated with serving)